rdt-open-reader/src/test/java/io/ona/rdt/robolectric/widget/WidgetFactoryRobolectricTest.java

```diff
@@ -33,19 +33,27 @@ public void tearDown() {
     }
 
     public static RDTJsonFormActivity getRDTJsonFormActivity() throws JSONException {
+        return getRDTJsonFormActivity(new JSONObject());
+    }
+
+    public static RDTJsonFormActivity getRDTJsonFormActivity(JSONObject stepObject) throws JSONException {
         return Mockito.spy(Robolectric.buildActivity(RDTJsonFormActivity.class,
-                getJsonFormActivityIntent())
+                getJsonFormActivityIntent(stepObject))
                 .create()
                 .resume()
                 .get());
     }
 
-    private static Intent getJsonFormActivityIntent() throws JSONException {
+    private static Intent getJsonFormActivityIntent(JSONObject stepObject) throws JSONException {
         JSONObject mJSONObject = new JSONObject();
-        mJSONObject.put(JsonFormConstants.STEP1, new JSONObject());
+        mJSONObject.put(JsonFormConstants.STEP1, stepObject);
```

Comment:
Call `getStepObject()` here.

Reply:
i can't call `getStepObject()` here because `getJsonFormActivityIntent()` method is static and `getStepObject()` is non-static

Reply:
`getStepObject()` can also be made static.

Reply:
if i made this static then i can't override this method in `CovidRepeatingGroupFactoryTest.class`

Reply:
You can

Reply:
Declaring a method with the same signature will mask the parent class static method

Reply:
I have checked but still not working even change the signature `protected` to `public`